### ISSUE TITLE
Some things cannot have their TRANSLUCENCY flag removed in DECOHACK

### ISSUE THREAD

**Issue by HALFKOOL** [MTrop/DoomTools#112]
```
#include <boom>
#include <friendly>
 
thing MTF_SOULSPHERE
{
	-TRANSLUCENT
}
thing MTF_INVULNERABILITY
{
	-TRANSLUCENT
}
thing MTF_MEGASPHERE
{
	-TRANSLUCENT
}
thing MTF_PARTIALINVISIBILITY
{
	-TRANSLUCENT
}
```
Running the above code in DECOHACK will output a blank dehacked file. Currently all of the power spheres have this issue.

**Comment by MTrop** [comment]
I think you got the missing ones - the rest have the TRANSLUCENT flag set. Had to verify in the Boom code.

Quick fix.

**Comment by MTrop** [comment]
Fixed in commit 9d953d84c783a92866f42af22271dbdd94605760.
